README.md

```diff
@@ -19,23 +19,23 @@
 
 **Fast:** Yarn caches every package it downloads so it never needs to download the same package again. It also parallelizes operations to maximize resource utilization so install times are faster than ever.
 
-**Reliable:** Using a detailed, but concise, lockfile format, and a deterministic algorithm for installs, Yarn is able to guarantee that an install that worked on one system will work exactly the same way on any other system.
+**Reliable:** Thanks to a detailed (but concise) lockfile format and a deterministic algorithm for installs, Yarn is able to guarantee that an install that worked on one system will work exactly the same way on any other system.
 
 **Secure:** Yarn uses checksums to verify the integrity of every installed package before its code is executed.
 
 ## Features
 
-* Offline Mode: If you've installed a package before, you can install it again without any internet connection.
-* Deterministic: The same dependencies will be installed the same exact way across every machine regardless of install order.
-* Network Performance: Yarn efficiently queues up requests and avoids request waterfalls in order to maximize network utilization.
-* Multiple Registries: Install any package from either npm or Bower and keep your package workflow the same.
-* Network Resilience: A single request failing won't cause an install to fail. Requests are retried upon failure.
-* Flat Mode: Resolve mismatching versions of dependencies to a single version to avoid creating duplicates.
-* More emojis. 🐈
+* **Offline Mode.** If you've installed a package before, you can install it again without any internet connection.
+* **Deterministic.** The same dependencies will be installed in the same exact way on any machine, regardless of install order.
+* **Network Performance.** Yarn efficiently queues up requests and avoids request waterfalls in order to maximize network utilization.
+* **Multiple Registries.** Install any package from either npm or Bower. Keep your package workflow the same.
```

Comment:
The change here seems reasonable, but if @thejameskyle thinks it should be removed, then let's remove it.